report_thesis/src/sections/summary.tex

```diff
@@ -0,0 +1,62 @@
+This technique involves firing high-powered lasers at soil samples to create plasma, which emits light that is captured by spectrometers aboard the rovers.
+The resulting spectra can be analyzed using machine learning models to determine the presence and concentration of major oxides, providing insights into Mars' geology.
+However, predicting oxide compositions from \gls{libs} data presents significant computational challenges due to the data's high dimensionality, non-linearity, and multicollinearity.
+Additionally, the high cost of data collection often results in small datasets, making it difficult to build accurate and robust models.
+
+\vspace{0.5em}
+
+Previous research has attempted to improve predictions using regression techniques and dimensionality reduction methods, enhancing both accuracy and interpretability.
+However, the complex, nonlinear nature of \gls{libs} data remains a significant challenge, necessitating more adaptive and robust machine learning strategies.
+Our thesis aims to build upon previous work by developing a machine learning pipeline tailored to the unique characteristics of \gls{libs} data, aiming for higher prediction accuracy and robustness.
+
+\vspace{0.5em}
+
+To achieve these objectives, we systematically explore ten different machine learning models identified through extensive literature review and the consideration of unconventional approaches.
+These models fall into three categories: ensemble learning models, linear and regularization models, and neural network models.
+Additionally, we investigate various preprocessing techniques, including scaling, dimensionality reduction, and data transformation.
+We developed a customized k-fold data partitioning algorithm to ensure rigorous evaluation and prevent data leakage.
+This method involved assigning fold numbers sequentially using a modulo operation for a random-like distribution and handling extreme values by redistributing them evenly across the training folds.
+Additionally, we managed extreme concentration values by identifying them at specific percentiles and ensuring they were distributed evenly across the training folds, preventing any single fold from being disproportionately influenced.
+We created a web application with a slider to determine the percentile value for handling extreme values and dropdown menus to select the target oxide and the cross-validation method, which would then plots to visualize the distribution of extreme values across the folds.
+Our cross-validation framework systematically evaluated model performance using these partitions, providing robust estimates of accuracy and generalizability.
+To identify the most effective combinations of models and preprocessing techniques, we employed an automated hyperparameter optimization framework, Optuna, which systematically searched for optimal hyperparameters for each regression target.
```

Comment:
Does this make it sound like we relied solely on Optuna? I think it does a bit. Would like to highlight all the work we put in as well :)